tests/unittests/execution_cache/decorators.py

```
@@ -0,0 +1,62 @@
+"""
+Decorator utils for unit tests.
+
+| Copyright 2017-2025, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+from functools import wraps
+import platform
```

Ruff (0.8.2)

`platform` imported but unused

Remove unused import: `platform`

(F401)